cli/cmd/root.go

```diff
@@ -107,5 +107,24 @@ func Execute() error {
 			log.Warn(warning)
 		}
 	}
+
+	WarnIfHappyOutdated(rootCmd)
+
 	return nil
 }
+
+func WarnIfHappyOutdated(cmd *cobra.Command) {
+
+	outdated, cliVersion, latestAvailableVersion, err := IsHappyOutdated(cmd)
+
+	if err != nil {
+		log.Errorf("Error checking for latest available version number: %v", err)
+		return
+	}
+
+	if outdated {
+		log.Warnf("This copy of Happy CLI is not the latest availble. Please update at your earliest convenience.")
+		log.Warnf("CLI version: %s  Latest available version: %s", cliVersion.Version, latestAvailableVersion.Version)
```

Comment:
Can we print the command used to update?

`brew upgrade happy`

Reply:
Sure! Good idea!

Reply:
Now reads:

```
[WARNING]: This copy of Happy CLI is not the latest available. CLI version: 999.12.34-SNAPSHOT-26b60f8  Latest available version: v0.64.0

[WARNING]: To update on Mac, run:  brew upgrade happy
```